src/core/scenario.jl

```diff
@@ -129,7 +129,8 @@ function Scenario(d::Dict; flex_hvac_from_json=false)
                                             off_grid_flag=settings.off_grid_flag,
                                             time_steps_per_hour=settings.time_steps_per_hour,
                                             analysis_years=financial.analysis_years,
-                                            load_year=electric_load.year
+                                            load_year=electric_load.year,
+                                            clean_energy_fraction_series = Float64[]
```

Comment:
Do these need to be here? This is typically used to pass inputs from other Structs

Reply:
Corrected this. Thanks